drizzlepac/hapsequencer.py

```diff
@@ -74,6 +74,8 @@
 from stwcs import wcsutil
 from stwcs.wcsutil import headerlet
 
+import pdb
```

Comment:
This should not remain in operational code.

Reply:
Sorry, code was still "In progress".  This line of code is deleted.